src/index.js

```diff
@@ -339,8 +294,7 @@ export default class HyperScreen extends React.Component {
    *   used to render the screen.
    * Returns a promise that resolves to a DOM element.
    */
-  fetchElement = (href, verb, root, formData) => {
-    verb = verb || 'GET';
+  fetchElement = async (href, method, root, formData) => {
     if (href[0] === '#') {
       return new Promise((resolve, reject) => {
```

Comment:
This should not be a promise, especially if you've converted the wrapping function to `async`. It doesn't look it needs to be a promise at all actually since there's nothing asynchronous happening.

Reply:
addressed in 746f667